src/js/components/Institutions.jsx

```diff
@@ -177,7 +183,13 @@ export default class Institution extends Component {
           period="2017"
           userName={this.props.user.profile.name} />
         <div className="usa-width-two-thirds">
-          {this.props.filings.map((filingObj, i) => {
+          {this.props.isFetching || !this.props.filings ?
+            <div className="usa-grid-full">
+              <p style={{visibility: 'hidden'}}>Loading data</p>
```

Comment:
Possibly, not the `<p style={{visibility: 'hidden'}}>Loading data</p>` part alone, cause otherwise the width would collapsed. Probably should just adjust the styles for the` LoadingIconWrapper`.